package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "LBRY",
-  "version": "0.21.2",
+  "version": "0.20.0",
```

Comment:
Guessing you didn't mean to add this

Reply:
This was done by me in order to test the modal. Has to be undone, naturally.